### ISSUE TITLE
Add column to books to indicate official vs. homebrew content

### ISSUE THREAD

**Issue by duffn** [duffn/theouterrim#15]
https://www.reddit.com/r/swrpg/comments/f6nyyh/introducing_the_outer_rim_swrpg_stats_site_and/fi8qzzu?utm_source=share&utm_medium=web2x

If we allow addition of homebrew content to the site, we should add a column to indicate if a book is official or homebrew.

**Comment by bknowles** [comment]
For the Viluppo index, it was a very conscious decision on Jake’s part not to allow any home brew items. And to specifically avoid including any of the item description fluff. I did add a bunch of pictures to some things, mostly based on images I could find on Wookieepedia.

I think these are the kinds of decisions that you want to think long and hard about, then clearly and publicly document whichever way you decide.

**Comment by duffn** [comment]
Thank you for the input. Do you have enough information to expand on that decision? What was the reasoning to not include homebrew items?

**Comment by Tailball** [comment]
As stated in the pull request: it is ready but is awaiting your decision to include homebrew or not.

**Comment by bknowles** [comment]
> Thank you for the input. Do you have enough information to expand on that decision? What was the reasoning to not include homebrew items?

I think the biggest problem is that most home-brew items are way better than anything else in the system. Even if you follow the crafting rules properly, it’s possible to build a Blaster Rifle that does more damage than any Heavy Repeating Blaster type in the game, but is Encumbrance 1, and through multiple Triumphs has a bunch of other stuff integrated that you simply couldn’t imagine anyone would ever want anything else. I know, because I have a character with a blaster rifle he calls “Angry Cricket”. I can kill Krayt Dragons with it in one round. Sure, it could be a Unique item in the game, just like Cratala’s lightsaber. But it is so OP compared to all other FFG gear that I think even that would be a mistake.

I think it’s much easier to just say “No Homebrew, period.”

OTOH, there is some homebrew that I think is actually well designed and not OP compared to FFG gear. In the book “Spark of Rebellion”, we tried very hard to make sure that every single item we introduced was definitely not OP. We chose benchmark items from FFG, and made sure that nothing we built would be better than those benchmark items in all or most respects. Sure, our item might be better in one or two respects, but then it would be worse in others, and so we would overall maintain the balance.

But if you took a “No homebrew, period” approach, then you couldn’t include anything from “Spark of Rebellion”.

So, I’m not sure what the right answer is here.

I am convinced that you want to make a decision that you will feel comfortable defending in the future. Because no matter what decision you make, you will have to defend it.
